builtin/providers/aws/resource_aws_route53_zone.go

```diff
@@ -171,6 +171,10 @@ func resourceAwsRoute53ZoneRead(d *schema.ResourceData, meta interface{}) error
 		d.Set("delegation_set_id", cleanDelegationSetId(*zone.DelegationSet.Id))
 	}
 
+	if zone.HostedZone != nil && zone.HostedZone.Config != nil && zone.HostedZone.Config.Comment != nil {
```

Comment:
@jen20 there is no real risk here. It can be made an empty string but I just wanted to catch any potential panic points :)